spark-3.1-spanner-lib/src/main/java/com/google/cloud/spark/spanner/SpannerScanner.java

```diff
@@ -87,13 +89,13 @@ public InputPartition[] planInputPartitions() {
     String sqlStmt = selectPrefix + " FROM " + this.spannerTable.name();
     Filter[] filters = this.getFilters();
     if (filters.length > 0) {
-      sqlStmt += " WHERE " + SparkFilterUtils.getCompiledFilter(true, filters);
+      sqlStmt +=
+          " WHERE "
+              + SparkFilterUtils.getCompiledFilter(
+                  true, DataFormat.AVRO, Optional.empty(), filters);
```

Comment:
Please remove the DataFormat dependencies.

Reply:
Done!